Open_Source_Events/README.md

```diff
@@ -5,4 +5,6 @@ List of Open Source Events in which we have participated.
 - PClub Summer of Code (PSoC)
 - STUDENT CODE-IN (SCI)
 - Contributor's Hack 2020 (CH20)
-- Semester Long Projects (SLoP)
\ No newline at end of file
+- Semester Long Projects (SLoP)
+- Kharagpur Winter of Code (KWOC)
+- NJack Winter of Code (NWOC)
```

Comment:
It should be NJACK and not NJack, I guess

Reply:
Okay !!